sklearn/preprocessing/imputation.py

```diff
@@ -248,6 +251,14 @@ def _sparse_fit(self, X, strategy, missing_values, axis):
 
                 return most_frequent
 
+            elif strategy == "random":
```

Comment:
You should also update the comment above `# Median + Most frequent`, and add a comment here to be consistent.